core/templates/services/date-time-format.service.ts

```diff
@@ -51,8 +51,11 @@ export class DateTimeFormatService {
     return dayjs(date).format('MMM D hh:mm A');
   }
   // Returns just the date.
-  getLocaleDateString(millisSinceEpoch: number): string {
-    let date = new Date(millisSinceEpoch);
+  getLocaleDateString(millisSinceEpoch: number | null): string {
```

Comment:
Do we actually use `getLocaleDateString(null)` anywhere in our codebase (except for tests/spec files)?

Reply:
Hi @vojtechjelinek ,  we call `getLocaleDateString(null) ` in dateTimeFormatService test currently.

Reply:
Okay, then remove the test with `null` and only allow `number` in this method.

Reply:
Sure, that sounds good! I will remove it.